src/app/series/series.component.spec.ts

```diff
@@ -30,13 +30,13 @@ describe('SeriesComponent', () => {
 
   cit('loads a series by id', (fix, el, comp) => {
     activatedRoute.testParams = {id: '99'};
-    auth.mock('prx:series', {id: 99, title: 'my series title'});
+    auth.mock('prx:series', {id: 99, title: 'my series title'}).mock('prx:account', {id: 78});
     fix.detectChanges();
     expect(el).toContainText('my series title');
     expect(comp.series.id).toEqual(99);
   });
 
-  cit('creates new series from the default account', (fix, el, comp) => {
+  cit('defaults new series to the default account', (fix, el, comp) => {
     activatedRoute.testParams = {};
     auth.mock('prx:default-account', {id: 88});
```

Comment:
Rewording this test is probably misleading. Honestly probably could have left the existing wording because it is still technically true. This isn't really where it uses multiple accounts. For New Series, series component will only have the default account and is used to instantiate the series, which is where it gets the default, but the multiple accounts come into play on the series basic component, which calls back to the series component to setSeries with the alternate account.